src/pages/policies/policies.tsx

```diff
@@ -70,25 +86,69 @@ interface PoliciesLayoutProps {
 }
 
 function PoliciesLayout(props: PoliciesLayoutProps) {
+  const params = useQueryParams();
   const history = useHistory();
+
   const handleFilterChange = (showActive: boolean, showInactive: boolean) => {
+    const newParams = new URLSearchParams(params);
     if (showActive && !showInactive) {
-      history.replace(`/policies?filter=active`);
+      newParams.set('filter', 'active');
     } else if (!showActive && showInactive) {
-      history.replace(`/policies?filter=inactive`);
+      newParams.set('filter', 'inactive');
     } else if (!showActive && !showInactive) {
-      history.replace(`/policies?filter=none`);
+      newParams.set('filter', 'none');
     } else {
-      history.replace(`/policies`);
+      newParams.delete('filter');
     }
+    history.replace('/policies?' + newParams.toString());
   };
 
-  const params = useQueryParams();
   const filter = params.get('filter');
   const activeChecked = !filter || filter === 'active';
   const inactiveChecked = !filter || filter === 'inactive';
+
+  const query = params.get('ipfs-hash') || '';
+  const handleSubmit: FormEventHandler<HTMLFormElement> = (ev) => {
+    ev.preventDefault();
```

Comment:
I think this gets a bit too subjective haha. Personally I don't think one can argue for or against newlines in specific places, because it's one of those things that is too fuzzy. I know a dev with close to 20 years under his belt, and he personally dislikes too many newlines, as it reduces the amount of code you can read without the need to scroll. 

But I wouldn't ever leave feedback for fuzzy things like this, it's in the eye of the beholder.

Reply:
Yeah, that's why I always consider them as "nits". Feel free to NOT implement them if you disagree.